packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md

```diff
@@ -52,29 +52,29 @@ Okta event hooks also require a response. To prevent unnecessary delays or timeo
 * Respond immediately to the HTTP request with either a 200 (Success) or 204 (Success no content) return code.
 * Process the event hook request data after sending the response code.
 
-A timeout of three seconds is enforced on all outbound requests for event and inline hooks, with one retry in the event of a timeout or an error response from the external service. If a successful response has not been received after that, a 400 error is returned with more information about what failed.
+A timeout of three seconds is enforced on all outbound requests for event and inline hooks. One request retry is sent in the event of a timeout or an error response from the external service. If a successful response is not received after that, a 400 error is returned with more information about what failed.
 
-See inline hook [Timeout and Retry](/docs/concepts/inline-hooks/#timeout-and-retry) and event hook [Timeout and Retry](/docs/concepts/event-hooks/#timeout-and-retry).
+See inline hook [Timeout and retry](/docs/concepts/inline-hooks/#timeout-and-retry) and event hook [Time out and retry](/docs/concepts/event-hooks/#timeout-and-retry).
 
 ## Limits, duplicates, and order of hook calls
 
 The number of hook calls and the limits per org are available in the following table. Keep in mind these numbers and limits when designing your hook solution.
 
-Your external service processing hook requests must take into consideration that the order of event or inline hook calls is not guaranteed. Also, to avoid processing duplicate requests, use the `eventId` property to identify unique requests.
+Your external service that processes hook requests must consider the order of event or inline hook calls isn't guaranteed. Also, to avoid processing duplicate requests, use the `eventId` property to identify unique requests.
```

Comment:
```suggestion
Your external service that processes hook requests must consider that the order of events or inline hook calls aren't guaranteed. Also, to avoid processing duplicate requests, use the `eventId` property to identify unique requests.
```